MIGRATING-0.6-0.7.md

```diff
@@ -1,26 +1,29 @@
 # Migrating from embedded-graphics 0.6.x to 0.7.0
 
+> Please note that this migration guide may be incomplete in some sections. If any missing or incorrect information is found, please [open an issue](https://github.com/embedded-graphics/embedded-graphics/issues/new) or [join the Element chatroom](https://matrix.to/#/#rust-embedded-graphics:matrix.org) to bring it to our attention.
```

Comment:
Nit:
```suggestion
> Please note that this migration guide may be incomplete in some sections. If any missing or incorrect information is found, please [open an issue](https://github.com/embedded-graphics/embedded-graphics/issues/new) or [join the Matrix chatroom](https://matrix.to/#/#rust-embedded-graphics:matrix.org) to bring it to our attention.
```
But otherwise I like it.